cluster/gce/trusty/master.yaml

```diff
@@ -216,7 +216,6 @@ script
 	export HOME="/root"
 	export KUBECTL_BIN="/usr/bin/kubectl"
 	export TOKEN_DIR="/etc/srv/kubernetes"
-	export kubelet_kubeconfig_file="/var/lib/kubelet/kubeconfig"
```

Comment:
If the entire block in kube-addons.sh is deleted, this export can be removed also.